Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -522,6 +527,44 @@ Messages::WebDriverClient::GetTitleResponse WebDriverConnection::get_title()
     return title;
 }
 
+// 11.5 New Window, https://w3c.github.io/webdriver/#dfn-new-window
+Messages::WebDriverClient::NewWindowResponse WebDriverConnection::new_window()
+{
+    // 1. If the implementation does not support creating new top-level browsing contexts, return error with error code unsupported operation.
+
+    // 2. If the current top-level browsing context is no longer open, return error with error code no such window.
+    TRY(ensure_open_top_level_browsing_context());
+
+    // 3. Handle any user prompts and return its value if it is an error.
+    TRY(handle_any_user_prompts());
+
+    // FIXME: 4. Let type hint be the result of getting the property "type" from the parameters argument.
+
+    // 5. Create a new top-level browsing context by running the window open steps with url set to "about:blank",
+    // 5. target set to the empty string, and features set to "noopener" and the user agent configured to create a new
+    // 5. browsing context. This must be done without invoking the focusing steps for the created browsing context. If
+    // 5. type hint has the value "tab", and the implementation supports multiple browsing context in the same OS
+    // 5. window, the new browsing context should share an OS window with the current browsing context. If type hint
+    // 5. is "window", and the implementation supports multiple browsing contexts in separate OS windows, the
+    // 5. created browsing context should be in a new OS window. In all other cases the details of how the browsing
+    // 5. context is presented to the user are implementation defined.
```

Comment:
I meant like this:

```
// 5. xxx
//    xxx
//    xxx
...
```

Reply:
I misunderstood comment in a goofiest way, sorry 😅 Fixed!